client/src/components/chat/chat.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * Created by einavcarmon on 19/06/2017.
+ */
+
+import React, { Component } from 'react';
+import { createChat } from '../../api';
+import { Tabs, Tab } from 'react-bootstrap';
+import io from 'socket.io-client'
+
+import { Modal, Button } from 'react-bootstrap'
+import Strings from '../../strings'
+import { BackToFrequentBtn } from '../common';
+
+import ChatTab from "./chatTab";
+import './chat.css'
+
+export default class chatTab extends Component {
+
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            chats : [],
+            client: props.user ? props.user.firstName : '',
+            activeChatIndex: 0,
+            showModal:true,
+            user: props.user
+        };
+
+        // todo move url to .env (@dancar help ?)
```

Comment:
Did you have a look at how we implemented env-vars in analytics.js? no reason it shouldn't work the same I reckon